performant-software/DM
# Project menu scroll, "referenced by" wording change, non-admin form edit disabling

### What does this PR do?
Addresses a group of front-end element-styling and text changes:
- Adds a height limit and overflow scrolling to the dropdown Projects menu
- Changes "targets" to "referenced by" and "no targets" to "referenced by none" in the annotations popup
- Disables interaction with form elements in the project settings modal for users without admin permissions

### What issues does it address?
Closes #146, closes #151, closes #152 

### How to test
- Visit [staging](http://dm-staging.middell.net/) and hard-reload to update cached CSS
- Open the dropdown projects menu. If enough projects appear in the menu to make the container reach its maximum height, you should be able to scroll down in the list to see the full set of projects.
- As an admin for a project, open the project settings and set another account's permissions to include read and write but not admin. Log out and sign in as this other account, then open the project. Open the settings and try to change the value of the title, description, and public-checkmark elements (you should be unable to). Close the project settings, click "New project" and confirm that you can still edit the title and description fields in the new project dialog.
